ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/aaa-authentication.j2

```diff
@@ -22,4 +22,11 @@ aaa authentication policy on-success log
 {%     if aaa_authentication.policies.local.allow_nopassword is arista.avd.defined(true) %}
 aaa authentication policy local allow-nopassword-remote-login
 {%     endif %}
+{%     if aaa_authentication.policies.lockout.failure is arista.avd.defined and aaa_authentication.policies.lockout.duration is arista.avd.defined %}
+{%         set lockout_cli = "aaa authentication policy lockout failure " ~ aaa_authentication.policies.lockout.failure ~ " duration " ~ aaa_authentication.policies.lockout.duration %}
```

Comment:
this code is not super clear. I would first add the common part. Then do the test for the optional window command and then add the duration last. This way you don't have the same stuff multiple times.